scripts/rust/test-sbf.mjs

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env zx
 import 'zx/globals';
 import {
-  cliArguments,
-  workingDirectory,
+    cliArguments,
+    workingDirectory,
 } from '../utils.mjs';
 
 const [folder, ...args] = cliArguments();
-const manifestPath = path.join(workingDirectory, folder, 'Cargo.toml');
+const mainProgramManifestPath = path.join(workingDirectory, folder, 'Cargo.toml');
+const testProgramManifestPath = path.join(workingDirectory, folder, 'tests', 'helpers', 'test-transfer-hook', 'Cargo.toml');
 
-await $`RUST_LOG=error cargo test-sbf --manifest-path ${manifestPath} ${args}`;
+await $`RUST_LOG=error cargo test-sbf --manifest-path ${testProgramManifestPath} ${args}`;
+await $`RUST_LOG=error cargo test-sbf --manifest-path ${mainProgramManifestPath} ${args}`;
```

Comment:
Important to add the test program here as the main program tests depend on the .so file